build.gradle

```diff
@@ -171,6 +176,17 @@ task buildAwsLc {
         if (file(awslcSrcPath).list().size() == 0) {
             throw new GradleException("aws-lc dir empty! run 'git submodule update --init --recursive' to populate.")
         }
+
+        if (!isLegacyBuild) {
```

Comment:
Again, why do we have to do this for the non-FIPS build as well?

Reply:
We do this only for non-Legacy builds. There is only one legacy build that tests ACCP with an old version GCC using an Ubuntu 10.04 image.